### DISCUSSION TITLE
Mozilla Firefox URL line invisible while typing URL, menu overlay with search entries

### DISCUSSION THREAD

**Discussion by shiftF12ATwebDOTde** [Aris-t2/CustomCSSforFx#678] [General]
problem solved

thank you Aris
solution is very simple.
downloaded actual vrsion of userchrome files here on github from Aris
finsihed
solution takes 1 minute

hello.
since the last two versions of firefox i have the described problem.
i attach 2 screenshots.
the URL line is visible when i am not typing in it.
in the moment i click in it it still visible.
the moment i start to write in it, all kind of information blends over what i am writing and i can not see what i am typing into the URL line.

in the one picture you see the possible settings for the URL line. and ONLY when i click away all and everything, the URL line stays visible for me while i am tpying.

that is a really bad error and i have no idea why nobody complains.
may be i am one of only a few who experience this error.

i use a lot of UserChrome settings, but i never had any problems with these like this here.
![bild1](https://github.com/user-attachments/assets/c24083fe-37ae-43fd-bb06-c2bd71272be2)
![bild2](https://github.com/user-attachments/assets/ae28323f-4832-4ee9-be47-c8b7438e1f41)

when i click away everything and i type in a new URL i can see what i type.
see here
![bild3](https://github.com/user-attachments/assets/0ccaad87-6234-4f91-94e3-3d9cc3b04a7e)

but, when i then try to change the URL i again can not see what i am typing

see here
![bild4](https://github.com/user-attachments/assets/98b272ae-6c8e-4225-9319-e93b8b1f76da)

would be great if someone could help.

i reported this to Mozilla but i think they didnt read my email.

**Comment by Aris-t2** [comment]
Are you using this projects latest files? If yes, please attach your userChrome.css file (zipped).

**Comment by shiftF12ATwebDOTde** [comment]
oh my god

the solution can be so simple

 

THANK YOU

 

hahahahahahaha

 

problem solved

 

damn

I am an idiot

hahahahhahahaaa

 

 

 

Von: Aris ***@***.***> 
Gesendet: Montag, 16. September 2024 17:11
An: Aris-t2/CustomCSSforFx ***@***.***>
Cc: shiftF12ATwebDOTde ***@***.***>; Author ***@***.***>
Betreff: Re: [Aris-t2/CustomCSSforFx] Mozilla Firefox URL line invisible while typing URL, menu overlay with search entries (Discussion #678)

 

Are you using this projects latest files? If yes, please attach your userChrome.css file (zipped).

**Comment by shiftF12ATwebDOTde** [comment]
solved
see explanation on top
